benchmarks/benchmarks.py

```diff
@@ -75,3 +75,44 @@ def time_get_parts_python(self):
             parts.append(pygeos.get_geometry(self.multipolygons[i], range(num_parts)))
 
         parts = np.concatenate(parts)
+
+
+class STRtree:
+    """Benchmarks queries against STRtree"""
+
+    def setup(self):
+        self.polygons = pygeos.polygons(np.random.random((100, 3, 2)))
+        self.tree = pygeos.STRtree(self.polygons)
+
```

Comment:
Maybe can do here a tiny query with a single point or so, to already trigger the tree creation